zokrates_js/tests/stdlib.test.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert');
+const utils = require('../js/utils');
+const stdlib = require('../stdlib.json');
+
+describe('stdlib', function() {
+
+    it('should resolve module from stdlib (1)', function() {
+        let basePath = 'hashes/sha256/512bitPacked.zok';
+        let relativePath = '../../utils/pack/pack128';
+
+        let absolutePath = utils.appendExtension(utils.getAbsolutePath(basePath, relativePath), '.zok');
+        assert.notEqual(stdlib[absolutePath], undefined);
```

Comment:
Here we're testing that it resolves to something, could we test that it resolves to the right thing? `utils/pack/pack128` I assume?

Reply:
Just saw that's the approach you take for non-stdlib, why doesn't it work here?

Reply:
@Schaeff how would you test it that it resolved to right thing?
Hardcoding content of pack128 in the test? (That doesn't make that much sense to me)
Also testing that path is okay is utils/pack/pack128 also doesn't make sense.